src/workerd/jsg/util.c++

```diff
@@ -123,6 +123,16 @@ bool setRemoteError(v8::Isolate* isolate, v8::Local<v8::Value>& exception) {
       v8::True(isolate)));
 }
 
+bool setDurableObjectResetError(v8::Isolate* isolate, v8::Local<v8::Value>& exception) {
+  KJ_ASSERT(exception->IsObject());
+  auto obj = exception.As<v8::Object>();
+  return jsg::check(
+    obj->Set(
+      isolate->GetCurrentContext(),
+      jsg::v8Str(isolate, "isDurableObjectReset"_kj, v8::NewStringType::kInternalized),
```

Comment:
The `is` prefix here seems a bit inconsistent with the `.remote` field (i.e. it's `.remote`, not `.isRemote`). I don't feel super strongly, but I think just `.durableObjectReset` would match better.

Reply:
Hmmmmm, I'd be curious as to @jasnell 's opinion here. I considered naming parity with `.remote` but `.durableObjectReset` didn't feel like a property name for a boolean to me.

Reply:
I agree with @a-robinson here. I'm not a fan of the `is*` prefix on getters for JavaScript types.

Reply:
I'll change it over!